java/junit.ui/src/org/netbeans/modules/junit/ui/resources/JUnit5Suite.java.template

```diff
@@ -0,0 +1,24 @@
+<#-- FreeMarker template (see http://freemarker.org/) -->
```

Comment:
Please add the template file to the [licenseinfo.xml](https://github.com/apache/incubator-netbeans/blob/master/java/junit.ui/licenseinfo.xml) if don't add the license header to template files. e.g. see https://github.com/apache/incubator-netbeans/blob/7b3b31b62e5fa95f7fbf6a60cbcc5b0838f42740/ide/xml/licenseinfo.xml#L23-L31

Reply:
Hi @junichi11 Thanks for spotting this.

Just a note on this point though, the other licenses aren't covered in a licenseinfo.xml file, they are infact covered via the rat exclusions file: https://github.com/apache/incubator-netbeans/blob/7b3b31b62e5fa95f7fbf6a60cbcc5b0838f42740/nbbuild/rat-exclusions.txt#L188. So my new template file should be covered by this, as opposed to a licenseinfo.xml entry.

I added a new logo to differentiate JUnit 5 and the old JUnit version so will definitely add a reference to that as that was missed and commit that shortly.

Reply:
I see. I suppose that we should move them to the licenseinfo.xml if possible because we have to verify rat-exclusions and licenseinfo.xml :)